### DISCUSSION TITLE
Missing "times" property in the "PingResponse" interface.

### DISCUSSION THREAD

**Discussion by ValuWaurld** [DefinitelyTyped/DefinitelyTyped#55759] [Issues with a @types package] [answered]
Hey, I noticed that the **"times"** property, which is **an Array of Number**, is missing in the **"PingResponse"** interface for the **"@types/ping"** _addon_ (I would say?).

Since it's a property that is received for one or multiple **"min_reply"**, I'm asking to add it.

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/9f7da13d13441ebf165622f4454b28b03e4c589f/types/ping/index.d.ts#L34-L71

![1_try](https://user-images.githubusercontent.com/34285426/132735232-eb5fcb9c-d05f-4d66-95a7-e5684b47e2e9.png)
![5_try](https://user-images.githubusercontent.com/34285426/132735238-9e06010f-c25b-44cd-a770-ba42cd4d4044.png)

[a comment by an automated account is not shown]

**Comment by BendingBender** [comment, marked as answer]
Updated `ping` types to v0.4: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/56579
